app/main.py

```diff
@@ -1,13 +1,14 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"source": "flake8", "line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"], "name": error["code"]}
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/216051512-0ff8bb44-ec59-4132-993a-076c60834a08.png)
fix this according to the best practicies, use line only for one key/value pair